code/modules/paperwork/paper_cutter.dm

```diff
@@ -6,108 +6,161 @@
+		if(isnull(user_head)) // So no head?
+			user.visible_message(span_suicide("[user] tries to behead [user.p_them()]self with [src], but [user.p_they()] [user.p_were()] already missing it! How embarassing!"))
+			return SHAME
+		user.visible_message(span_suicide("[user] is beheading [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+		user_head.drop_limb()
+		playsound(loc, SFX_DESECRATION, 50, TRUE, -1)
 		return BRUTELOSS
+	// If we have no blade, just beat ourselves up
+	user.visible_message(span_suicide("[user] repeatedly bashes [src] against [user.p_them()]self! It looks like [user.p_theyre()] trying to commit suicide!"))
+	playsound(loc, 'sound/items/gavel.ogg', 50, TRUE, -1)
+	return BRUTELOSS
 
 
 /obj/item/papercutter/update_icon_state()
-	icon_state = (storedcutter ? "[initial(icon_state)]-cutter" : "[initial(icon_state)]")
+	icon_state = (stored_blade ? "[initial(icon_state)]-cutter" : "[initial(icon_state)]")
 	return ..()
 
 /obj/item/papercutter/update_overlays()
 	. =..()
-	if(storedpaper)
+	if(stored_paper)
 		. += "paper"
 
 /obj/item/papercutter/screwdriver_act(mob/living/user, obj/item/tool)
-	if(!storedcutter)
+	if(!stored_blade && !blade_secured)
+		balloon_alert(user, "no blade to secure!")
 		return
 	tool.play_tool_sound(src)
-	to_chat(user, span_notice("[storedcutter] has been [cuttersecured ? "unsecured" : "secured"]."))
-	cuttersecured = !cuttersecured
+	balloon_alert(user, "blade [blade_secured ? "un" : ""]secured")
+	blade_secured = !blade_secured
 	return TOOL_ACT_TOOLTYPE_SUCCESS
 
 
-/obj/item/papercutter/attackby(obj/item/P, mob/user, params)
-	if(istype(P, /obj/item/paper) && !storedpaper && !istype(P, /obj/item/paper/paperslip))
-		if(!user.transferItemToLoc(P, src))
+/obj/item/papercutter/attackby(obj/item/inserted_item, mob/user, params)
+	if(istype(inserted_item, /obj/item/paper) && !istype(inserted_item, /obj/item/paper/paperslip))
+		if(stored_paper)
+			balloon_alert(user, "already paper inside!")
+		if(!user.transferItemToLoc(inserted_item, src))
 			return
 		playsound(loc, SFX_PAGE_TURN, 60, TRUE)
-		to_chat(user, span_notice("You place [P] in [src]."))
-		storedpaper = P
-		update_appearance()
-		return
-	if(istype(P, /obj/item/hatchet/cutterblade) && !storedcutter)
-		if(!user.transferItemToLoc(P, src))
+		balloon_alert(user, "paper inserted")
+		stored_paper = inserted_item
+
+	if(istype(inserted_item, /obj/item/hatchet/cutterblade))
+		if(stored_blade)
+			balloon_alert(user, "already a blade inside!")
 			return
-		to_chat(user, span_notice("You replace [src]'s [P]."))
-		P.forceMove(src)
-		storedcutter = P
-		update_appearance()
-		return
+		if(!user.transferItemToLoc(inserted_item, src))
+			return
+		balloon_alert(user, "blade inserted")
+		inserted_item.forceMove(src)
+		stored_blade = inserted_item
+
+	update_appearance()
+
 	..()
 
 /obj/item/papercutter/attack_hand(mob/user, list/modifiers)
-	. = ..()
 	if(.)
 		return
```

Comment:
This is useless here now since we aren't calling parent, delete this. (also, are you _really_ sure that we want to override attack_hand's parent like this?)